ci/pipeline

```diff
@@ -221,3 +236,45 @@ def jenkins(): Unit = {
   if(utils.isPullRequest) pr()
   else master()
 }
+
+/**
+ * Executes the Community Release which includes:
+ *  1. tarball with version details
+ *  2. release of docker with that version
+ *  3. git tag of version (pushed to server)
+ *
+ * @return Version and artifact description of Marathon build.
+ */
+@main
+def release(requestVersion: String, gitSha: String, latest: String = "False"): Unit = {
+
+  val releaseVersion = cleanVersion(requestVersion)
+  val tagVersion = s"v$releaseVersion"
+  println(s"Releasing version: $releaseVersion ")
+  %('git, "checkout", gitSha)
+  %('git, "tag", "-a", tagVersion, "-m", tagVersion)
+  val version = build()
+
+  uploadTarballPackagesToS3(version, s"releases/$releaseVersion")
```

Comment:
As I mentioned before, should this be `s"releases/$version"`?

Reply:
there is a lot of "versions" :) .   if using $version the tarball would land in `/releases/v1.6.0/tarball` and we want `releases/1.6.0/tarball`